src/style.css

```diff
@@ -1 +1,73 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  --main-color: #00acdc;
+}
+
+.link {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  text-transform: uppercase;
+  color: #000;
+  text-decoration: none;
+  position: relative;
+  padding: 22.5px 0;
+  align-items: end;
+  display: flex;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  left: 0;
+  top: 0;
+  background: #fff;
+  margin: auto;
```

Comment:
Replace 'width: 100%;' with 'max-width: 1200px;' in the .header class to limit the layout to the specified width.